timemachine/fe/single_topology.py

```diff
@@ -925,14 +925,47 @@ def combine_confs(self, x_a, x_b):
             Combined conformation
 
         """
-        # tbd: allow input to take lambda schedule and return an array for each value of lambda
-        assert x_a.shape == (self.mol_a.GetNumAtoms(), 3)
-        assert x_b.shape == (self.mol_b.GetNumAtoms(), 3)
+        return self._combine_confs_ab(x_a, x_b)
+
+    def _combine_confs_ab(self, x_a, x_b):
```

Comment:
Make these public or have combine_confs take an extra argument for direction?

Otherwise you'll need to call a private function from outside the class.

Reply:
agreed, the naming is confusing as hell too. I'll rename this explicitly to `combine_confs_lhs` and `combine_confs_rhs` to denote setting up the conformation for the left and right end-states.